### PR TITLE
submission abort button should not be visible to readers

### PR DESCRIPTION
**Pull request by davidangb**
DataBiosphere/firecloud-app#156 - the submission abort button should not be visible to readers, since readers do not have permission to abort.

-----

- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by davidangb** [comment]
unrelated test failures:
```
[info] *** 4 TESTS FAILED ***
[error] Failed: Total 123, Failed 4, Errors 0, Passed 119, Ignored 6
[error] Failed tests:
[error] 	org.broadinstitute.dsde.firecloud.test.previewmodal.PreviewSpec
[error] 	org.broadinstitute.dsde.firecloud.test.metadata.DataSpec
[error] 	org.broadinstitute.dsde.firecloud.test.analysis.MethodImportSpec
[error] (Test / testOnly) sbt.TestsFailedException: Tests unsuccessful
```

**Comment by davidangb** [comment]
* PreviewSpec passed in a manual run: https://fc-jenkins.dsp-techops.broadinstitute.org/job/swatomation-pipeline/9063/
* DataSpec passed in a manual run: https://fc-jenkins.dsp-techops.broadinstitute.org/job/swatomation-pipeline/9060/
* MethodImportSpec passed in a manual run: https://fc-jenkins.dsp-techops.broadinstitute.org/job/swatomation-pipeline/9061/

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/analysis/MethodLaunchSpec.scala [hunk 1]
```diff
@@ -218,4 +220,57 @@ class MethodLaunchSpec extends FreeSpec with ParallelTestExecution with Matchers
     }
   }
 
+  "reader cannot abort a launched submission" in {
```
Comment (jmthibault79) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/analysis/MethodLaunchSpec.scala]: nitpick: can we make this "does not see an abort button"

**Review by jmthibault79 — APPROVED** [review]

**Comment by davidangb** [comment]
3 commits ago, all tests passed in a combination of automation and manual runs.

The last two commits had 1 failure and 3 failures, but all failures were different - between the last two commits, all tests passed.

**Pull request merged**